integration_tests/array_01.py

```diff
@@ -4,7 +4,7 @@
 def main0():
     Nx: i32 = 600; Ny: i32 = 450
     arr: i32[450, 600] = empty([Ny, Nx])
-    i: i32
+    i: i32 = 42
```

Comment:
```suggestion
    i: i32 = 69
```

Might as well make it justifiable